v1/branch/branch.gunk

```diff
@@ -199,11 +204,16 @@ type BranchService interface {
 	// }
 	// +gunk openapiv2.Operation{
 	//         Tags:        []string{"Branches"},
-	//         Description: "Get All available branch.",
-	//         Summary:     "Get All available branch.",
+	//         Description: "Retrieve information regarding all available branches.",
+	//         Summary:     "Retrieve all available branches",
 	//         Responses: map[string]openapiv2.Response{
 	//                 "200": openapiv2.Response{
 	//                         Description: "Request executed successfully.",
+	//                         Schema: openapiv2.Schema{
+	//                                 JSONSchema: openapiv2.JSONSchema{
+	//                                         Ref: "#/definitions/branchGetBranchesResponse",
```

Comment:
Should the Ref be "#/definitions/branch`es`GetBranchesResponse?

Reply:
see: https://github.com/openbank/openbank/pull/17#discussion_r314579504